libse/SubtitleFormats/Utx.cs

```diff
@@ -53,37 +56,37 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
         {
             _errorCount = 0;
             subtitle.Paragraphs.Clear();
-            var text = new StringBuilder();
+            var sb = new StringBuilder();
+            char[] splitChars = { '#', ',' };
+            char[] splitChars2 = { ':', '.' };
             for (int i = 0; i < lines.Count; i++)
             {
                 string line = lines[i].Trim();
-
-                if (line.StartsWith('#'))
+                if (line.StartsWith('#') && RegexTimeCode.IsMatch(line))
                 {
-                    var timeParts = line.Split(new[] { '#', ',' }, StringSplitOptions.RemoveEmptyEntries);
-                    if (timeParts.Length == 2)

```

Comment:
Will be handled inside `DecodeTimeCodeFramesFourParts` since `DecodeTimeCodeFramesFourParts` throws exception if parts aren't equal to 4 this will improve detecting error count (_errorCount)